django_project/ford3/models/qualification.py

```diff
@@ -129,6 +129,16 @@ def interest_id_list(self) -> List[int]:
             result.append(each_item['id'])
         return result
 
+    @property
+    def interest_name_list(self) -> List[int]:
+        result = []
+        interest_query = Interest.objects.filter(
```

Comment:
see list comprehension in python.
`result [each_item['name'] for each_item in list(interest_query)`